[2024-01-30T20:32:31.922+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-01-30T20:32:31.932+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-01-30T20:32:31.932+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-30T20:32:31.966+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): yellow_cabs_upload> on 2024-01-29 00:00:00+00:00
[2024-01-30T20:32:31.970+0000] {standard_task_runner.py:60} INFO - Started process 80 to run task
[2024-01-30T20:32:31.979+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'yellow_cabs_dag', 'yellow_cabs_upload', 'scheduled__2024-01-29T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/yellow_cabs_project.py', '--cfg-path', '/tmp/tmppu27y_hc']
[2024-01-30T20:32:32.425+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask yellow_cabs_upload
[2024-01-30T20:32:32.469+0000] {task_command.py:423} INFO - Running <TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [running]> on host 9921d23a6722
[2024-01-30T20:32:32.510+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='yellow_cabs_dag' AIRFLOW_CTX_TASK_ID='yellow_cabs_upload' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-29T00:00:00+00:00'
[2024-01-30T20:32:41.647+0000] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/yellow_cabs_project.py", line 42, in yellow_cabs_upload
    table.to_sql('yellow_cabs_table', con=db, if_exists='append', index=False)
AttributeError: 'pyarrow.lib.Table' object has no attribute 'to_sql'
[2024-01-30T20:32:41.655+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=yellow_cabs_dag, task_id=yellow_cabs_upload, execution_date=20240129T000000, start_date=20240130T203231, end_date=20240130T203241
[2024-01-30T20:32:41.661+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 5 for task yellow_cabs_upload ('pyarrow.lib.Table' object has no attribute 'to_sql'; 80)
[2024-01-30T20:32:41.743+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-01-30T20:32:41.751+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-30T20:44:53.172+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-01-30T20:44:53.184+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-01-30T20:44:53.184+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-30T20:44:53.200+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): yellow_cabs_upload> on 2024-01-29 00:00:00+00:00
[2024-01-30T20:44:53.208+0000] {standard_task_runner.py:60} INFO - Started process 72 to run task
[2024-01-30T20:44:53.215+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'yellow_cabs_dag', 'yellow_cabs_upload', 'scheduled__2024-01-29T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/yellow_cabs_project.py', '--cfg-path', '/tmp/tmpk234crvc']
[2024-01-30T20:44:53.222+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask yellow_cabs_upload
[2024-01-30T20:44:53.295+0000] {task_command.py:423} INFO - Running <TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [running]> on host 2710995934d7
[2024-01-30T20:44:53.379+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='yellow_cabs_dag' AIRFLOW_CTX_TASK_ID='yellow_cabs_upload' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-29T00:00:00+00:00'
[2024-01-30T20:45:07.587+0000] {local_task_job_runner.py:234} INFO - Task exited with return code -9
[2024-01-30T20:45:07.651+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-31T18:15:44.009+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-01-31T18:15:44.021+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-01-31T18:15:44.022+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-31T18:15:44.054+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): yellow_cabs_upload> on 2024-01-29 00:00:00+00:00
[2024-01-31T18:15:44.059+0000] {standard_task_runner.py:60} INFO - Started process 99 to run task
[2024-01-31T18:15:44.062+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'yellow_cabs_dag', 'yellow_cabs_upload', 'scheduled__2024-01-29T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/yellow_cabs_project.py', '--cfg-path', '/tmp/tmp66lxokwj']
[2024-01-31T18:15:44.065+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask yellow_cabs_upload
[2024-01-31T18:15:44.102+0000] {task_command.py:423} INFO - Running <TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [running]> on host 8ff05e883b9e
[2024-01-31T18:15:45.138+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='yellow_cabs_dag' AIRFLOW_CTX_TASK_ID='yellow_cabs_upload' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-29T00:00:00+00:00'
[2024-01-31T18:16:02.748+0000] {local_task_job_runner.py:234} INFO - Task exited with return code -9
[2024-01-31T18:16:02.952+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-31T21:58:10.681+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-01-31T21:58:10.708+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-01-31T21:58:10.708+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-31T21:58:10.722+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): yellow_cabs_upload> on 2024-01-29 00:00:00+00:00
[2024-01-31T21:58:10.727+0000] {standard_task_runner.py:60} INFO - Started process 74 to run task
[2024-01-31T21:58:10.740+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'yellow_cabs_dag', 'yellow_cabs_upload', 'scheduled__2024-01-29T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/yellow_cabs_project.py', '--cfg-path', '/tmp/tmpdbvar8xs']
[2024-01-31T21:58:10.752+0000] {standard_task_runner.py:88} INFO - Job 6: Subtask yellow_cabs_upload
[2024-01-31T21:58:11.134+0000] {task_command.py:423} INFO - Running <TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [running]> on host d6a8637ce3ab
[2024-01-31T21:58:11.170+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='yellow_cabs_dag' AIRFLOW_CTX_TASK_ID='yellow_cabs_upload' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-29T00:00:00+00:00'
[2024-01-31T21:58:41.513+0000] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/yellow_cabs_project.py", line 35, in yellow_cabs_upload
    df_iter = pd.read_csv('/Users/oluwaseyi/Desktop/YellowCabs_Project/csv_file.csv',
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 912, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 577, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1407, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1661, in _make_engine
    self.handles = get_handle(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/common.py", line 859, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/Users/oluwaseyi/Desktop/YellowCabs_Project/csv_file.csv'
[2024-01-31T21:58:41.523+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=yellow_cabs_dag, task_id=yellow_cabs_upload, execution_date=20240129T000000, start_date=20240131T215810, end_date=20240131T215841
[2024-01-31T21:58:41.537+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 6 for task yellow_cabs_upload ([Errno 2] No such file or directory: '/Users/oluwaseyi/Desktop/YellowCabs_Project/csv_file.csv'; 74)
[2024-01-31T21:58:41.626+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-01-31T21:58:41.635+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-02T19:22:04.334+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-02-02T19:22:04.457+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [queued]>
[2024-02-02T19:22:04.464+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-02-02T19:22:04.588+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): yellow_cabs_upload> on 2024-01-29 00:00:00+00:00
[2024-02-02T19:22:04.622+0000] {standard_task_runner.py:60} INFO - Started process 82 to run task
[2024-02-02T19:22:04.641+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'yellow_cabs_dag', 'yellow_cabs_upload', 'scheduled__2024-01-29T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/yellow_cabs_project.py', '--cfg-path', '/tmp/tmp7qvely73']
[2024-02-02T19:22:04.657+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask yellow_cabs_upload
[2024-02-02T19:22:04.795+0000] {task_command.py:423} INFO - Running <TaskInstance: yellow_cabs_dag.yellow_cabs_upload scheduled__2024-01-29T00:00:00+00:00 [running]> on host 66814aa8a277
[2024-02-02T19:22:11.728+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='yellow_cabs_dag' AIRFLOW_CTX_TASK_ID='yellow_cabs_upload' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-29T00:00:00+00:00'
[2024-02-02T19:22:52.344+0000] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/yellow_cabs_project.py", line 35, in yellow_cabs_upload
    df_iter = pd.read_csv('/Users/oluwaseyi/Desktop/YellowCabs_Project/csv_file.csv',
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 912, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 577, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1407, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1661, in _make_engine
    self.handles = get_handle(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/common.py", line 859, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/Users/oluwaseyi/Desktop/YellowCabs_Project/csv_file.csv'
[2024-02-02T19:22:52.495+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=yellow_cabs_dag, task_id=yellow_cabs_upload, execution_date=20240129T000000, start_date=20240202T192204, end_date=20240202T192252
[2024-02-02T19:22:52.739+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 9 for task yellow_cabs_upload ([Errno 2] No such file or directory: '/Users/oluwaseyi/Desktop/YellowCabs_Project/csv_file.csv'; 82)
[2024-02-02T19:22:52.813+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-02T19:22:59.167+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
